app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/TransferTypeButtons.kt

```diff
@@ -42,7 +45,9 @@ import com.infomaniak.swisstransfer.ui.utils.PreviewLightAndDark
 @Composable
 fun TransferTypeButtons(transferType: GetSetCallbacks<TransferTypeUi>) {
     Row(
-        modifier = Modifier.horizontalScroll(rememberScrollState()),
+        modifier = Modifier
+            .horizontalScroll(rememberScrollState())
+            .windowInsetsPadding(insets = WindowInsets(left = Margin.Medium, top = 0.dp, right = Margin.Medium, bottom = 0.dp)),
```

Comment:
Why use a `windowInsetsPadding` over a `padding`?

Reply:
I was convinced that adding padding wasn't working because the padding wasn't scrollable, I guess I was just running old version of code, it works indeed